src/i18n/en.json

```diff
@@ -42,8 +42,8 @@
     "sendDetail": "Send Details",
     "sendAddressError": "Please enter a valid wallet address",
     "amountError": "The format of the amount you entered is incorrect, please check and re-enter",
-    "confirmSendingToValidatorDescription": "This is a validator wallet address. Transfers to this address do not stake your funds with the validator.",
-    "confirmSendingToValidatorTitle": "Are you sure you want to continue?",
+    "confirmTransferringToValidator": "This is a validator wallet address. Transfers to this address do not stake your funds with the validator.",
```

Comment:
with this change, this string is used both for transfers and paratime withdrawals. the string specifically says transfers though

Reply:
Umm, it's still only used for transfers

Reply:
oh what did removing this condition do? https://github.com/oasisprotocol/oasis-wallet-ext/pull/236/commits/739cc24e3a05d1b9c75f1fd89e8c9440855eeaaf#diff-f6f1bab03cecf5a59db7c6104df694c9120dc25405d0342bc8f1876b3ccdaf8fL568

Reply:
oh that's to make it allow other warn functions run, but we still only set it for transfers